MailLibrary/Drivers/IDriver.php

```diff
@@ -134,10 +134,23 @@ function copyMail($mailId, $toMailbox);
 	 */
 	function moveMail($mailId, $toMailbox);
 
+
+	/**
+	 * Save mail content as eml to given file
+	 * @param int $mailId
+	 * @param $file
+	 * @throws DriverException
+	 */
+	function saveMail($mailId, $file);
```

Comment:
path to file does not seem to me as good approach se we are assuming that e-mail is writable filesystem; I would use stream or just string with e-mail mime-message content.